android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -211,9 +227,12 @@ private void onClickViews() {
         if (mBtnPositive != null) {
             mBtnPositive.setOnClickListener(
                     view -> {
-                        if (mCurrentStep == 1 && !isDefaultBrowser()) {
+                        // If this is the first step and Brave is not set as default browser
+                        if (mCurrentStep == 0 && !isDefaultBrowser()) {
+                            // Show default browser prompt and proceed to next step
```

Comment:
The comment says literally the exact same thing as the method invocation. I would remove it.